lms/djangoapps/commerce/tests.py

```diff
@@ -0,0 +1,248 @@
+""" Tests for commerce views. """
+
+import json
+from uuid import uuid4
+
+from ddt import ddt, data
+from django.core.urlresolvers import reverse
+from django.test.utils import override_settings
+import httpretty
+from httpretty.core import HTTPrettyRequestEmpty
+import jwt
+from requests import Timeout
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from commerce.constants import OrderStatus
+from course_modes.models import CourseMode
+from enrollment.api import add_enrollment
+from student.models import CourseEnrollment
+from student.tests.factories import UserFactory, CourseModeFactory
+
+
+ECOMMERCE_API_URL = 'http://example.com/api'
+ECOMMERCE_API_SIGNING_KEY = 'edx'
+ORDER_NUMBER = 2008
+ECOMMERCE_API_SUCCESSFUL_BODY = json.dumps({'status': OrderStatus.COMPLETE, 'number': ORDER_NUMBER})
+
+
+@ddt
+@override_settings(ECOMMERCE_API_URL=ECOMMERCE_API_URL, ECOMMERCE_API_SIGNING_KEY=ECOMMERCE_API_SIGNING_KEY)
+class OrdersViewTests(ModuleStoreTestCase):
+    """
+    Tests for the commerce orders view.
+    """
+
+    def _login(self):
+        """ Log into LMS. """
+        self.client.login(username=self.user.username, password='test')
+
+    def _post_to_view(self, course_id=None):
+        """
+        POST to the view being tested.
+
+        Arguments
+            course_id (str) --  ID of course for which a seat should be ordered.
+
+        :return: Response
+        """
+        course_id = unicode(course_id or self.course.id)
+        return self.client.post(self.url, {'course_id': course_id})
+
+    def _mock_ecommerce_api(self, status=200, body=None):
```

Comment:
In this case, yes. The use of DRF is an implementation detail. I'd prefer to keep the tests framework-agnostic. Ideally, if we decided to scrap DRF, these tests should still be valid.